bootstrap.js

```diff
@@ -4,42 +4,49 @@
 
 Components.utils.import("resource://gre/modules/Services.jsm");
 
-function startup(data, reason)
-{
-  Services.prefs.setBoolPref("dom.vr.enabled", true);
+const PREFS = {
+  // Enables WebVR. Not needed for Nightly, but we need to keep for other versions.
+  "dom.vr.enabled": true,
```

Comment:
`gfx.vr.mirror-textures = true`
see: https://bugzilla.mozilla.org/show_bug.cgi?id=1196435

`dom.vr.add-test-devices = 1`
If we don't set this, VR does not work unless HMD is plugged in.   

I feel that these should both be set, but I know @kearwood may disagree (and reasons for this)